cloudpickle/cloudpickle.py

```diff
@@ -238,7 +238,11 @@ def dump(self, obj):
 
     def save_memoryview(self, obj):
         """Fallback to save_string"""
-        Pickler.save_string(self, str(obj))
+        try:
+            arg = obj.obj
+        except ValueError:
+            arg = _released_memory_view_sentinel
+        self.save_reduce(_fill_memoryview, (arg,), obj=obj)
```

Comment:
The only clean way to pickle a memoryview would be to recreate the `PyObject_GetBuffer` call when unpickling, and then recreate the various operations (indexing, casting) which were made on the memoryview object...